Feat/add scs 0005 project governance (#813)
SovereignCloudStack/standards: Standards/scs-0005-v1-project-governance.md

---

## Introduction

The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
for a range of cloud infrastructure types as well as a modular open-source
reference implementation.
The project is governed by the _SCS Project Board_.

## Role of the _SCS Project Board_

The role of the _SCS Project Board_ is the overall governance of the SCS Community and Project.
This happens together with the _Forum SCS-Standards_ of the Open Source Business Alliance. To further
underline this alignment, the _Forum SCS-Standards_ is part of the _SCS Project Board_.
The _SCS Project Board_ itself is elected by the _SCS Community_.

### Definitions

#### _SCS Project_

The _SCS Project_ is the Open-Source project that consists of the software, documentation, documents, blog posts as well as the people ("_SCS Community_") working on this.

#### _SCS Community_

The collective of people, companies, and organizations promoting the idea of the _SCS Project_ as well as the people working on the various aspects.

## Election of the _SCS Project Board_

### Term

The _SCS Project Board_ is elected for the term of one year. Elections are done

> in November. The December serves as a transition period.

Author: > Should we add here "starting with November 2025"? Since November 2024 is already over and we still need to do a voting this year?

I addressed this with a better wording (I think ;)

### Seats on the board

The _SCS Project Board_ contains five seats. One of these seats is filled by
the delegate of the _Forum SCS-Standards_. The other four seats are voted upon.

### Nominations

Every person who is part of the Sovereign Cloud Stack GitHub organization can be
nominated for the board. Likewise, one can nominate oneself.

### Eligible for voting

> Every person who is a member of the GitHub organization "Sovereign Cloud Stack" is eligible for voting.

Author: Good point. I suggest that the github-manager repo (which manages the org) is then maintained by the project board. How about that?

> Every person who is a member of the GitHub organization "Sovereign Cloud Stack" is eligible for voting.

Reviewer: Another proposal:
If you want to join the org, you **either** can be accepted through application at the project board **or** find two active community members that vouch for you.
Thoughts?

### Electoral management

The voting process is governed by the _Forum SCS-Standards_.

### Mechanisms

Each voting person has eight (8) points that can be cast during the voting process.

> The points can be distributed between four candidates.

Author: @berendt this mechanism came from you. Please elaborate and clarify directly in the file.

> The points can be distributed between four candidates.

Author: From my understanding "the points can be distributed between four candidates" means: yes, you can also allocate all of them on one candidate. Wether or not this is what @berendt had in mind, when he suggested this: 🤷🏻‍♂️ 

I was the one who suggested using heliosvoting.org for this, since that is a system well-known to work (we've previously used it at the OSBA for votings). However heliosvoting.org does not support this point-based voting (eg. one of course can enter the points into a textfield, but the system does not technically assist in wether this done correctly or not).

@berendt do you have a suggestion on how this could be done in a way that you envisioned?

> The points can be distributed between four candidates.

Reviewer: I brought this point up, because I think the voting mechanism is the most important part of the document and it not being clearly specified might lead to unwanted outcomes in the future.
We can absolutely not clarify further on "how to distribute" and just accept any way that is within the bounds. We should however still clarify how the actual voting process is done. Is it a secret/public voting? Is it done in the same document that handles nominations or via third party tool (there is probably more open questions that I can not immediately think about)?
Should we not specify anymore, the "specification work" and the "how are we doing this" is just put upon the shoulders of the current and future forum members (assuming this is what you meant with "The voting process is governed by the Forum SCS-Standards.").
Would be nice to get your opinion on this @berendt :)

> The points can be distributed between four candidates.

Reviewer 2: Very interesting questions! We should use different pages/tools for nominations and voting. So, we have the challenges that the tool should allow the point giving voting, the results should be anonymously, but it must also ensure that a person can only vote once... 

What needs to be more specific? @michaelbayr

The four candidates with the highest number of points overall are then elected.

## Roles in the _SCS Project Board_

> Among the elected Project Board a spokesperson is nominated.

Author: Please add your proposal to the file directly.

> Among the elected Project Board a spokesperson is nominated.

Author: I've added a suggestion based on your comment.

This happens together with the _Forum SCS-Standards_ of the Open Source Business Alliance. To further
underline this alignment, the _Forum SCS-Standards_ is part of the _SCS Project Board_.
The _SCS Project Board_ itself is elected by the _SCS Community_.

### Definitions

#### _SCS Project_

The _SCS Project_ is the Open-Source project that consists of the software, documentation, documents, blog posts as well as the people ("_SCS Community_") working on this.

#### _SCS Community_

The collective of people, companies, and organizations promoting the idea of the _SCS Project_ as well as the people working on the various aspects.

### _SCS GitHub Organization_

The _SCS GitHub Organization_ is this: [https://github.com/sovereigncloudstack](https://github.com/sovereigncloudstack)

## Joining the SCS GitHub Organization

Since being part of the GitHub organization comes with a set of responsibilities, joining the SCS GitHub Organization can be done by:

- being invited by the _SCS Project Board_
- submitting a request to be onboarded as a member to the _SCS Project Board_
- have to existing members of the GitHub organization that nominate you

> One of these items is sufficient.

Reviewer 3: ```suggestion
One of these items is sufficient.

Actively contributing to one or several of the projects under the governance of the SCS project board should typically result in a membership. Please be aware of our [Code of Conduct](https://github.com/SovereignCloudStack/.github/blob/main/CODE_OF_CONDUCT.md).
```

Reviewer 3: I want to express that we github org membership is a result of the contribution, not a consequence of happening to know an existing member. The latter is the mechanism, not the selection criteria.
This is up for discussion of course, so I won't just push this myself.

Reviewer: I think this is a good addition. 👍

### Eligible for voting

Every person who is a member of the GitHub organization "Sovereign Cloud Stack" is eligible for voting. In order to be able to vote an onboarding onto the Identity Management of the SCS community needs to happen.

### Electoral management

The voting process is governed by the _Forum SCS-Standards_.
Voting is done using the [Condorcet Internet Voting Service](https://civs.cs.cornell.edu). This is the same system as is [being used by the OpenInfra foundation](https://wiki.openstack.org/wiki/Election_Officiating_Guidelines#Running_the_election_itself).

### Voting period

The voting will be open for a week.

### Announcement

The voting will be announced on the SCS-Members Mailinglist as well as on the [General & Announcements](https://matrix.to/#/#scs-general:matrix.org). Enlisted voters will receive e-mails to the email address used in the SCS community's Identity Management system.

### Mechanisms

> Each eligble voter is asked to rank the canidates according to their priorities.

Reviewer 3: ```suggestion
Each eligible voter is asked to rank the candidates according to their priorities.
```

status: Draft
track: Global
description: |
  SCS-0005 outlines the structure and governance of the SCS community by the SCS Project Board and how this is elected.
---

## Introduction

The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
for a range of cloud infrastructure types as well as a modular open-source
reference implementation.
The project is governed by the _SCS Project Board_.

## Role of the _SCS Project Board_

The role of the _SCS Project Board_ is the overall governance of the SCS Community and Project.
This happens together with the _Forum SCS-Standards_ of the Open Source Business Alliance. To further
underline this alignment, the _Forum SCS-Standards_ is part of the _SCS Project Board_.
The _SCS Project Board_ itself is elected by the _SCS Community_.

### Definitions

#### _SCS Project_

The _SCS Project_ is the Open-Source project that consists of the software, documentation, documents, blog posts as well as the people ("_SCS Community_") working on this.

#### _SCS Community_

The collective of people, companies, and organizations promoting the idea of the _SCS Project_ as well as the people working on the various aspects.

> ### _SCS GitHub Organization_
> 
> The _SCS GitHub Organization_ is this: [https://github.com/sovereigncloudstack](https://github.com/sovereigncloudstack)
> 

Reviewer 4: ```suggestion
#### _SCS GitHub Organization_

The _SCS GitHub Organization_ is this: [https://github.com/sovereigncloudstack](https://github.com/sovereigncloudstack)

### Roles in the _SCS GitHub Organization_

#### Members

Joining the SCS GitHub Organization as a contributor results in being assigned the **member role** in the organization. Members are contributors or collaborators who:

- Actively contribute to projects within the organization.
- Have repository-specific access based on their contributions.
- Are eligible to vote in elections and nominate candidates for the _SCS Project Board_.
- Must adhere to the [Code of Conduct](https://github.com/SovereignCloudStack/.github/blob/main/CODE_OF_CONDUCT.md).

#### Owners

Members of the _SCS Project Board_ are also designated as **owners** of the SCS GitHub organization. Owners have administrative privileges, including:

- Managing organization-level settings.
- Onboarding new members.
- Enforcing compliance with governance and community standards.

This alignment ensures that governance roles in the SCS Project Board directly translate into operational responsibilities within the GitHub organization.

```
